docs/recipes/typescript.md

```diff
@@ -0,0 +1,56 @@
+# TypeScript
+
+AVA comes bundled with a TypeScript definition file. This allows developers to leverage TypeScript for writing tests.
+
+## Setup
+
+First install the TypeScript compiler [tsc](https://github.com/Microsoft/TypeScript).
+
+```
+$ npm install tsc --save-dev
+```
+
+Create a [`tsconfig.json`](https://github.com/Microsoft/TypeScript/wiki/tsconfig.json) file. This file specifies the compiler options required to compile the project or the test file.
+
+```json
+{
+	"compilerOptions": {
+		"module": "commonjs",
+		"target": "es2015"
+	}
+}
+```
+
+Add a `test` script in the `package.json` file. It will compile the project first and then run AVA.
+
+```json
+{
+  "scripts": {
+    "test": "tsc && ava"
+  }
+}
+```
+
+
+## Add tests
+
+Create a `test.ts` file.
```

Comment:
Does AVA actually find `.ts` files without it being explicitly defined? I'm not sure to be honest.